### DISCUSSION TITLE
Support Site Speed Sample Rate for Google Analytics

### DISCUSSION THREAD

**Discussion by ccjenniferhsu** [grafana/grafana#38286] [Dashboards, Panels & Visualizations]
**What would you like to be added**:
Support [siteSpeedSampleRate](https://developers.google.com/analytics/devguides/collection/analyticsjs/field-reference#siteSpeedSampleRate) in google analytics setting( https://github.com/grafana/grafana/blob/main/public/app/core/services/analytics.ts#L33 ).

**Why is this needed**:
Google analytics is very useful to track UX for Grafana. However, after enabling the [feature](https://grafana.com/docs/grafana/latest/administration/configuration/#google_analytics_ua_id), we found most of the average loading time is 0, and the reason is the default `siteSpeedSampleRate` is 1%. In current [analytics.ts ](https://github.com/grafana/grafana/blob/main/public/app/core/services/analytics.ts#L33 ), this is not configurable. I would like to ask if we can either change the default to 100 (ex: set to`ga('create', (config as any).googleAnalyticsId, {'siteSpeedSampleRate': 10});`) or make it configurable. Thanks!
![image](https://user-images.githubusercontent.com/40581391/119394516-1ee71680-bc87-11eb-948c-a6713523e67d.png)

**Comment by pkolyvas** [comment]
We're in the process of moving Grafana feature requests to [GitHub Discussions](https://community.grafana.com/t/were-moving-grafana-feature-requests-to-github-discussions/50341/3). This issue is being moved to [Discussions](https://github.com/grafana/grafana/discussions) or has been moved from our issues list, depending on where you're seeing this message. 

 We hope to use the discussion thread to continue building use cases, and additional evidence based on engagement/voting, in order to help us prioritize this feature request later on down the road. 

As with all things Grafana, we value your input and participation in this project and appreciate your patience during this transitional period. Looking forward to the discussion!

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
